### ISSUE TITLE
Firefox fails to reconnect websocket connection after server side timeout

### ISSUE THREAD

**Issue by Artur-** [Atmosphere/atmosphere#1938]
Combination required: 
* Atmosphere 2.2.6
* Jetty 9 (this will not happen with Jetty 8)
* Firefox (will not happen with other browsers)

Steps required:
* Add org.atmosphere.websocket.maxIdleTime=5000 to Chat sample
* mvn clean install jetty:run
* Open http://localhost:8080/ and wait 5s

Expected:
* Server closes idle connection, Atmosphere fires onReconnect, browser reconnects

Actual:
* Chrome, works as expected
* Firefox, does not fire onReconnect

Apparently this has changed in 2.3.0.RC6 and neither browser fires onReconnect. Is this an intentional change? If so, how can you get the reconnect behavior of Chrome/Atm 2.2.6?

**Comment by jfarcand** [comment]
@Artur- Let me try to understand. *Both* Chrome and Firefox aren't working with `2.3.0-RC6`?

**Comment by jfarcand** [comment]
Which version of the `atmosphere-javascript` are you using?

**Comment by Artur-** [comment]
Both Chrome and Firefox show the same behavior with 2.3.0-RC6, i.e. onClose is fired but not onReconnect

Javascript version is 2.2.9

**Comment by jfarcand** [comment]
OK seems the browser doesn't like receiving
```js
atmosphere.js:3220 Wed Apr 15 2015 11:54:40 GMT-0400 (EDT) Atmosphere: websocket.onclose
2015-04-15 11:54:40.654 atmosphere.js:3220 Websocket closed, reason: Idle Timeout
2015-04-15 11:54:40.654 atmosphere.js:3220 Websocket closed, wasClean: true
2015-04-15 11:54:40.654 atmosphere.js:3220 Wed Apr 15 2015 11:54:40 GMT-0400 (EDT) Atmosphere: Firing onClose
```

**Comment by jfarcand** [comment]
OK same issue for both browsers with 2.2.7-SNAPSHOT. The regression is from #1894

**Comment by jfarcand** [comment]
Closing, the issue is fixed vua Atmosphere/atmosphere-javascript#161
